core/services/fluxmonitorv2/validate_test.go

```diff
@@ -129,32 +129,6 @@ ds1 -> ds1_parse;
 				c.Set("DEFAULT_HTTP_TIMEOUT", "2s")
 			},
 		},
-		{
-			name: "async=true should error",
```

Comment:
Why has this gone?

Reply:
validation is just done at a higher level now